.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: node_js
-node_js: "stable"
 
-before_install:
-  - stty cols 80
-  - npm install
+before_script:
+  - npm run lint
```

Comment:
This `before_script` step means that it runs ESLint before executing tests. I figured this was the right move since there's no point in waiting until the entire test suite wraps up to find out that you have a linting error! But happy to be challenged on this.

Docs: https://docs.travis-ci.com/user/job-lifecycle/#breaking-the-build

Reply:
Yeah, I agree!!